batch7crud-roll4/src/main/java/com/lftechnology/batch7crud/dao/UserDao.java

```diff
@@ -0,0 +1,48 @@
+package com.lftechnology.batch7crud.dao;
+
+import com.lftechnology.batch7crud.exception.DataException;
+import com.lftechnology.batch7crud.model.Employee;
+import com.lftechnology.batch7crud.model.User;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.lftechnology.batch7crud.model.User'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)